executor/analyze.go

```diff
@@ -148,6 +175,11 @@ func (e *AnalyzeExec) analyzeWorker(taskCh <-chan *analyzeTask, resultCh chan<-
 			}
 		}
 	}
+	e.wg.Done()
```

Comment:
If the worker panics, then the following code won't be executed.

Reply:
Yes. I'll fix it.